developer_guide/installation/system_requirements/setup_requirements_ubuntu_1404.rst

```diff
@@ -0,0 +1,117 @@
+Setup System Requirements on Ubuntu 14.04
+==========================================
+
+Here is a quick guide to setup the :doc:`system_requirements` requirements on Ubuntu 14.04.
+
+System installation
+-------------------
+
+Base dependencies
+*****************
+
+Before installing requirements, update repositories information. You'll have to install MySQL, Apache and PHP.
+
+.. code-block:: bash
+    :linenos:
+
+    $ sudo apt-get update
+    $ sudo apt-get install mysql-server
+    $ sudo apt-get install apache2
+    $ sudo apt-get install libapache2-mod-php5 php5-cli
+    $ sudo apt-get install php5-mysql php5-intl php5-curl php5-gd php5-mcrypt
+    $ sudo php5enmod mcrypt
+    $ sudo apt-get install php5-apcu
+
+.. note::
+    PHP 5.5 provided in Ubuntu 14.04 comes with the Zend OPcache opcode cache. Only the data cache provided by APCu is needed.
+
+Choosing the product storage
+****************************
+
+.. include:: /reference/technical_information/choose_database.rst.inc
+
+Based on this formula, either you need :ref:`installing-mongodb`, either you can directly go to the :ref:`system-configuration` section.
+
+.. _installing-mongodb:
+
+Installing MongoDB
+******************
+
+* Install MongoDB server
+
+.. code-block:: bash
+    :linenos:
+
+    $ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv 7F0CEB10
+    $ sudo echo deb http://downloads-distro.mongodb.org/repo/debian-sysvinit dist 10gen | sudo tee /etc/apt/sources.list.d/mongodb-10gen.list > /dev/null
+    $ sudo apt-get update
+    $ sudo apt-get install mongodb-10gen=2.4.14
+
+.. note::
+
+    Akeneo PIM may work with versions 2.6 of MongoDB but not with 3.*.
+    To avoid updates, you can freeze the version with, for apt/dpkg: ``sudo echo "mongodb-10gen hold" | sudo dpkg --set-selections`` or for aptitude: ``sudo aptitude hold mongodb-10gen``
+
+* Install MongoDB PHP driver
+
+.. code-block:: bash
+    :linenos:
+
+    $ sudo apt-get install php5-mongo
+
+
+.. _system-configuration:
+
+System configuration
+--------------------
+
+MySQL
+*****
+
+* Create a MySQL database and a user for the application
+
+.. code-block:: bash
+    :linenos:
+
+    $ mysql -u root -p
+    mysql> CREATE DATABASE akeneo_pim;
+    mysql> GRANT ALL PRIVILEGES ON akeneo_pim.* TO akeneo_pim@localhost IDENTIFIED BY 'akeneo_pim';
+    mysql> EXIT
+
+PHP
+***
+
+.. include:: /reference/technical_information/php_ini.rst.inc
+
+Apache
+******
+To avoid spending too much time on permission problems between the CLI user and the Apache user, an easy configuration
+is to use the same user for both processes.
+
+* Get your identifiers
+
+.. code-block:: bash
+    :linenos:
+
+    $ id
+    uid=1000(my_user), gid=1000(my_group), ...
+
+In this example, the user is *my_user* and the group is *my_group*.
+
+* Use your identifiers for Apache
+
+.. code-block:: bash
+    :linenos:
+
+    $ sudo service apache2 stop
+    $ sudo gedit /etc/apache2/envvars
+    export APACHE_RUN_USER=my_user
```

Comment:
is it permanent. I think that exported values are reset for each new terminal sessiosn